### ISSUE TITLE
F-Droid reproducible build failed

### ISSUE THREAD

**Issue by linsui** [project-kaat/gpsdRelay#9]
https://gitlab.com/fdroid/checkupdates-bot-fdroiddata/-/jobs/8526219457 Your apk is not built from the tag. Please remove the vcs info or make sure the apk is built after the the changes are committed. See https://f-droid.org/en/docs/Reproducible_Builds/#vcs-info.

**Comment by project-kaat** [comment]
> https://gitlab.com/fdroid/checkupdates-bot-fdroiddata/-/jobs/8526219457 Your apk is not built from the tag. Please remove the vcs info or make sure the apk is built after the the changes are committed. See https://f-droid.org/en/docs/Reproducible_Builds/#vcs-info.

Updated the release apk

**Comment by linsui** [comment]
We find that there is a `DependencyInfoBlock` in your APK.

It's a [Signing block](https://bi-zone.medium.com/easter-egg-in-apk-files-what-is-frosting-f356aa9f4d1) added by AGP and encrypted with the Google public key so it can't be read by anyone else except Google. You can read more about it [here](https://gitlab.com/fdroid/admin/-/issues/367) and [here](https://gitlab.com/fdroid/fdroidserver/-/issues/1056).

While this was added a while ago, we were only enforcing it for new apps, and recently we started scanning updates too.

Could you please disable it with the following code?

```
android {
    dependenciesInfo {
        // Disables dependency metadata when building APKs.
        includeInApk = false
        // Disables dependency metadata when building Android App Bundles.
        includeInBundle = false
    }
}
```

Thanks!

**Comment by project-kaat** [comment]
> We find that there is a `DependencyInfoBlock` in your APK.
> 
> It's a [Signing block](https://bi-zone.medium.com/easter-egg-in-apk-files-what-is-frosting-f356aa9f4d1) added by AGP and encrypted with the Google public key so it can't be read by anyone else except Google. You can read more about it [here](https://gitlab.com/fdroid/admin/-/issues/367) and [here](https://gitlab.com/fdroid/fdroidserver/-/issues/1056).
> 
> While this was added a while ago, we were only enforcing it for new apps, and recently we started scanning updates too.
> 
> Could you please disable it with the following code?
> 
> ```
> android {
>     dependenciesInfo {
>         // Disables dependency metadata when building APKs.
>         includeInApk = false
>         // Disables dependency metadata when building Android App Bundles.
>         includeInBundle = false
>     }
> }
> ```
> 
> Thanks!

Done in 4d541231663628550c559814676e677882323c81.
I assume it's ok to save this change for the next release?

**Comment by licaon-kter** [comment]
how far is that into the future? :)

**Comment by project-kaat** [comment]
> how far is that into the future? :)

I was planning to do a UI redesign soon. With any luck, the next version should be out by the end of next week

**Comment by licaon-kter** [comment]
great

**Comment by licaon-kter** [comment]
testing now...

**Comment by licaon-kter** [comment]
`ERROR: Could not build app io.github.project_kaat.gpsdrelay: supplied reference binary signed with d27ba8e564dbd8ee8499d3146b8485d2049ebd8c7a76e0f157de5a5751654d82 instead of with ['9a44b31a2219369b2672d30d95204f3b95cfbe243a683c698f9b226e0ca3d511']`

@project-kaat did you lose your key? 😢

**Comment by project-kaat** [comment]
> `ERROR: Could not build app io.github.project_kaat.gpsdrelay: supplied reference binary signed with d27ba8e564dbd8ee8499d3146b8485d2049ebd8c7a76e0f157de5a5751654d82 instead of with ['9a44b31a2219369b2672d30d95204f3b95cfbe243a683c698f9b226e0ca3d511']`
> 
> [@project-kaat](https://github.com/project-kaat) did you lose your key? 😢

yeah. it was resolved in https://github.com/project-kaat/gpsdRelay/issues/7 though
...I hope 😅

**Comment by licaon-kter** [comment]
Ah, forgot about that, sad, eh

**Comment by licaon-kter** [comment]
https://gitlab.com/fdroid/fdroiddata/-/merge_requests/18922 thanks
